actors/cache/actors_cache.go

```diff
@@ -123,9 +123,19 @@ func (a *ActorsCache) ClearBadAddressCache() {
 }
 
 func (a *ActorsCache) GetActorCode(add address.Address, key filTypes.TipSetKey, onChainOnly bool) (string, error) {
```

Comment:
This logger line, on the lines below, is now (and has been) out of sense for forced onchain requests.

a.logger.Debugf("[ActorsCache] - Unable to retrieve actor code from offchain cache for address %s. Trying on-chain cache", add.String())